### ISSUE TITLE
Ideas for the Future of the Mod

### ISSUE THREAD

**Issue by ShotCosine** [brotazoa/Alloygery#14]
The mod is looking good so far. Some ideas I came up with for development:

- Changing flint and steel recipe to actually have steel instead of iron, since it wouldn't really change game progression for the worse
- Support for tags for ingots to accept ingots added from other mods
- Silver! Would love to see silver weapons added with the new smithing anvil system - there are currently no other mods for fabric 1.18 that add silver weapons/armor
- Hilts & handles for swords/tools since hilts for swords in minecraft are made of metal
- New weapon types - this is a stretch but it would be cool to be able to craft daggers, longswords, rapiers, claymores, battleaxes, etc with the smithing anvil and new materials - something that could be easier though could be adding support to globox_z's medieval weapons mod (https://www.curseforge.com/minecraft/mc-mods/medievalweapons)

Loving the mod so far, so much so that I might make a custom resource pack for it!

**Comment by brotazoa** [comment]
Thank you so much for the feedback.

More parts are definitely in the works (as you can see from the alpha builds) including handles, bindings, and additional tool heads. I like the idea of being able to craft new tool and weapon types, mining hammers, excavators, and such. Some thought would have to be put into the implementation of different weapons, in Vanilla there is a clear line between an axe and a sword, and there are reasons to choose one over the other in certain combat situations. So what kind of differences would a longsword, for example, have compared to a regular sword, and what kind of combat situations would one be more favorable. Maybe we can have a larger discussion on this topic.

Keep me in the loop on a resource pack, I will link to it on the curseforge page.

**Comment by ShotCosine** [comment]
I think that if this mod were to add its own new weapons to the game, most of them could be created by just removing/adding a certain part to a weapon. For example, making a sword without a hilt could create a machete-like weapon, using a thin blade could make a rapier, and using a long blade could create a longsword. I've seen the forge mod Tetra do this, automatically adding dozens of new weapons to the game. Tetra uses a very complicated system to create it's weapons though, maybe a simpler method could be achieved with this mod. 

I'm also interested in the idea of only using the smithing anvil to create weapons/armor. If this were added, then at least two more input slots should be added to the smithing anvil. This would allow the hilts and handles to be inputted to the anvil to choose from a list of possible weapons or tools. 

Another idea I thought of was increasing the materials required to make armor. In real life and almost every other game armor is made up of metal, leather, and bindings. In minecraft, armor is entirely made out of metal making it look kind of weird. If leather/binding is added to the smithing of armor then these can also be added cosmetically making the armor look miles better. If this was added then helmets should also wrap around the player's face like a real helmet. If I make a resource pack, I would texture the armor like this with leather and bindings, as well as improve the helmets - if these features are implemented in the future I could provide the textures to be added to the official mod.

What I love about this mod is that it combines many features into one - once I downloaded this I was able to delete 4 other mods - mythic metals, alloy forgery, and expert tools and armor. Keep up the good work!
